locales/fr/json.json

```diff
@@ -4,11 +4,11 @@
     "30 Days": "30 jours",
     "60 Days": "60 jours",
     "90 Days": "90 jours",
-    ":amount Total": ":Amount au total",
-    ":days day trial": ":Days jours d'essai",
+    ":amount Total": ":amount au total",
```

Comment:
There is no need to change these values because the script will automatically return them.

https://github.com/Laravel-Lang/common/discussions/7

Reply:
`:amount`, `:days` adn `trialDays` are numeric in Laravel Spark codebase, and capitalization doesn't seem necessary in that case.

Reply:
So there is no need to change them. In any case, our script will start the sentence with a capital letter.